package.json

```diff
@@ -35,6 +35,10 @@
     {
       "name": "Alex Smith",
       "email": "alex@gathercontent.com"
+    },
+    {
+      "name": "David Hollins",
+      "email": "david@gathercontent.com"
```

Comment:
🥳

Reply:
Amazing! Thanks for doing this, I feel like I kept meaning to do it then forgot.